manage.yml

```diff
@@ -36,8 +34,4 @@ groups:
       help_text: |
```

Comment:
Is there a reason you left UI group around? Other than "we might want to re-add UI, but with Airgun, in some indefinite future"? It doesn't seem to do anything.

Reply:
@mirzal, I am not sure. I am just starting to look into this code basis.

Reply:
if any of you @mirzal @JacobCallahan @rochacbruno think that any other change is required, I will update properly. I updated the min to get the ``manage shell`` working again.

Reply:
@kersommoura me neither :D . I didn't even know we have such thing in Robottelo until I saw your PR.

If you decide to spend more time on this and come to conclusion it's not needed, feel free to raise another PR.